src/components/WebexMeetingInfo/WebexMeetingInfo.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import jsonData from '../../data';
+import {WebexJSONAdapter} from '../../adapters';
+import {WebexMeetingInfo, WebexDataProvider} from '../';
+
+// Setup for the stories
+const stories = storiesOf('Webex Meeting Info', module);
+const webexAdapter = new WebexJSONAdapter(jsonData);
+const wrapperStyle = {height: '150px', width: '500px', border: '1px solid black'};
+
+stories.add('scheduled meeting', () => (
+  <div style={wrapperStyle}>
+    <WebexDataProvider adapter={webexAdapter}>
+      <WebexMeetingInfo meetingID="scheduledMeeting" />
+    </WebexDataProvider>
+  </div>
+));
+
+stories.add('one on one meeting', () => (
```

Comment:
In activity stream we have `1:1`
```suggestion
stories.add('1:1 meeting', () => (
```